### PR TITLE
Tune fixes

### PR DESCRIPTION
**Pull request by minad**
Tune works well, but there a a few minor issues:

* missing break in cmdline parsing
* missing gitignore files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,83 +1,86 @@
-# suppress compiler/linker output
```
Comment (minad) [inline .gitignore]: diff looks bad due to line ending changes. But you can select "hide whitespace changes" in the gh diff settings.

Comment (sjaeckel) [reply .gitignore]: ugh they were `\r\n`!?

btw. I added a separate etc/.gitignore
now it's mixed up where the stuff is ignored that is created in etc/ :-)

Comment (minad) [reply .gitignore]: I don't like multiple .gitignores tbh

File: .gitignore [hunk 2]
```diff
@@ -57,6 +57,7 @@ mpi.c
 
 # ignore file generated by make tune
 tuning_list
+etc/tune
```
Comment (minad) [inline .gitignore]: @sjaeckel I hope you agree. I find this simpler and it avoids confusion. I generally use single .gitignores and only multiple if there are multiple separate projects in a repository, like a monorepo with multiple vendored projects etc.

File: .gitignore (hunk 1 above)
Comment (sjaeckel) [reply .gitignore]: fine by me

File: .gitignore (hunk 2 above)
Comment (sjaeckel) [reply .gitignore]: shouldn't this then also be etc/tuning_list ?

Comment (minad) [reply .gitignore]: Yes, but I executed tune in . and then I also wanted tuning_list to be hidden. tuning_list seems to be a unique enough name to be ignored in the whole project I thought.

**Comment by czurnieden** [comment]
I missed a break? Really? Me? Please, don't tell the union! ;-)
Probably happened while cutting&pasting from`tune.c` of the faster_multiplication branch and I didn't test the "printing" function of the shorter one as thoroughly as it should have been tested, sorry for that.

Oh, and I found another one:
`   fprintf(stderr,"          -t prints comma separated results\n");`
Should read
`   fprintf(stderr,"          -t prints space (0x20) separated results\n");`
Not worth an extra PR, soooo if you wouldn’t mind? Please?

Thanks!

**Comment by sjaeckel** [comment]
>  Not worth an extra PR, soooo if you wouldn’t mind? Please?

done

**Comment by czurnieden** [comment]
> done

Thanks!

**Pull request merged**